### PR TITLE
Add reload button when the app errors out

### PR DESCRIPTION
**Pull request by licitdev**
## Scope

What's changed:

- Added reload button when the app errors out

image

## Potential Risks / Drawbacks

- Nil

## Review Notes / Questions

- Nil

---

Related to #22360 where the app errors out but without any action for users to proceed with

### CODE REVIEW

File: app/src/app.vue [hunk 1]
```diff
@@ -99,6 +103,8 @@ useSystem();
 
 			<template #append>
 				<v-error :error="error" />
+				<br />
+				<v-button kind="danger" @click="reload()">{{ t('reload') }}</v-button>
```
Comment (hanneskuettner) [inline app/src/app.vue:107]: Should this actually be a `danger` button or is `normal` fine?

Comment (hanneskuettner) [inline app/src/app.vue:107]: Drop the explicit call?

```suggestion
				<v-button kind="danger" @click="reload">{{ t('reload') }}</v-button>
```

Comment (paescuj) [reply app/src/app.vue]: Normal 👍

Comment (paescuj) [reply app/src/app.vue]: Could also be "Reload Page"

Comment (licitdev) [reply app/src/app.vue]: Changes were pushed via https://github.com/directus/directus/pull/22462/commits/4883f92c569efa7e91c948a70d095c057d29372c and https://github.com/directus/directus/pull/22462/commits/ae0b8c8f47c6f27e7e9ef669fec22c1ea1f3014c accordingly. Thank you both! 🚀

**Review by licitdev — COMMENTED** [review]
@paescuj The `unexpected_error_copy` is a generic message which is here: https://github.com/directus/directus/blob/6c287251367019f52cf80ca5b29e65ad1ea5f1c1/app/src/modules/content/routes/collection.vue#L504-L506

Should we add a new string `unexpected_error_reload` if we need to change it?

On the other hand, I think the "Try Again" button might be misleading as we are doing a reload and not retrying.

**Comment by paescuj** [comment]
> @paescuj The `unexpected_error_copy` is a generic message which is here:
> 
> https://github.com/directus/directus/blob/6c287251367019f52cf80ca5b29e65ad1ea5f1c1/app/src/modules/content/routes/collection.vue#L504-L506
> 
> Should we add a new string `unexpected_error_reload` if we need to change it?
> 
> On the other hand, I think the "Try Again" button might be misleading as we are doing a reload and not retrying.

Right. Hmm, I think it would already be sufficient if we drop the "later", so "Please try again". It feels wrong to tell the user to try later, while presenting a reload button at the same time.
This would also make sense for the case under `routes/collection.vue`.

**Review by paescuj — APPROVED** [review]
❤️

**Pull request merged**